scripts/base/frameworks/analyzer/main.zeek

```diff
@@ -149,6 +149,24 @@ export {
 	## Analyzer::register_for_port(s) and packet analyzers can add to this
 	## using PacketAnalyzer::register_for_port(s).
 	global ports: table[AllAnalyzers::Tag] of set[port];
+
+	## A set of protocol, packet or file analyzer tags requested to
+	## be enabled during startup.
+	##
+	## By default, all analyzers in Zeek are enabled. When all analyzers
+	## are disabled by :zeek:see:`Analyzer::disable_all`, this set
+	## set allows to record analyzers to be enabled during Zeek startup.
+	##
+	## This set can be added to via ``redef`` or calling :zeek:see:`Analyzer::request_analyzers`.
+	global requested_analyzers: set[AllAnalyzers::Tag] = {} &redef;
+
+	## Request the given analyzer tag to be enabled.
+	##
+	## Calling this method is only valid in :zeek:see:`zeek_init` handlers
+	## with a priority greater than -5.
+	##
+	## tags: The analyzer tags to request.
+	global request_analyzers: function(tags: set[AllAnalyzers::Tag]);
```

Comment:
Arne should we leave out this function? If requesting analyzers can only happen at startup, redef'ing `requested_analyzers` is actually the better mechanism, no?

Reply:
I'll drop in a fixup. The implementation that is being copied uses a function though. I pinged @sethhall for some input and then we squash or ditch the fixup.

Reply:
I just removed it without the extra commit. If there's follow-up feedback we can at the function back.